Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugSystemComponent.cpp

```diff
@@ -114,6 +116,29 @@ namespace Multiplayer
             ImGui::Checkbox("Multiplayer Entity Stats", &m_displayPerEntityStats);
             ImGui::Checkbox("Multiplayer Hierarchy Debugger", &m_displayHierarchyDebugger);
             ImGui::Checkbox("Multiplayer Audit Trail", &m_displayNetAuditTrail);
+
+            if (auto multiplayerInterface = AZ::Interface<IMultiplayer>::Get(); multiplayerInterface && !m_applicationType.IsEditor())
+            {
+                if (auto console = AZ::Interface<AZ::IConsole>::Get())
+                {
+                    const MultiplayerAgentType multiplayerAgentType = multiplayerInterface->GetAgentType();
+                    if (multiplayerAgentType == MultiplayerAgentType::Uninitialized)
+                    {
+                        if (ImGui::Button("Host"))
```

Comment:
nit: Move UX strings to static consts so we have some hope of localizing in the future?

Reply:
Fixed!